### PR TITLE
telco5g: Update cnf test step for 4-17 branching

### PR DESCRIPTION
**Pull request by Sandeepyadav93**
* Run 4.18 on cnfdu3
* Update test steps to support 4.18 jobs

[CNF-13943](https://issues.redhat.com/browse/CNF-13943)

### CODE REVIEW

**Comment by Sandeepyadav93** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-telco5g-cnftests

File: ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh [hunk 1]
```diff
@@ -382,7 +408,7 @@ fi
 pushd $CNF_REPO_DIR
 echo "******** Checking out pull request for repository cnf-features-deploy if exists"
 check_for_pr "openshift-kni" "cnf-features-deploy"
-if [[ "$T5CI_VERSION" == "4.15" ]] || [[ "$T5CI_VERSION" == "4.16" ]] || [[ "$T5CI_VERSION" == "4.17" ]]; then
+if [[ "$T5CI_VERSION" == "4.15" ]] || [[ "$T5CI_VERSION" == "4.16" ]] || [[ "$T5CI_VERSION" == "4.17" ]] || [[ "$T5CI_VERSION" == "4.18" ]]; then
```
Comment (sshnaidm) [inline ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:411]: maybe we can switch to condition when it's "not 4.12 and not 4.13 and not 4.14" so we stop updating it

**Comment by Sandeepyadav93** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-telco5g-cnftests

**Pull request merged**